src/components/AdminOrderPage/index.tsx

```diff
@@ -79,10 +36,27 @@ const AdminOrderPage: React.FC<AdminOrderPageProps> = (props) => {
         <Search
           placeholder="Search..."
           size="large"
-          // We'll include this console statement for now.
-          // We need a placeholder for the search functionality.
-          // eslint-disable-next-line no-console
-          onSearch={(value) => console.log(`Search not implemented! Here's value anyway: ${value}`)}
+          onSearch={(value) => {
+            // Search function, simply looks for substring (case-insensitive)
+            // within order user name and filters any order that don't have
+            // that name.
+            //
+            // If search bar is empty, just pass the original set of orders
+            // received from the API.
+            if (value === '') {
+              setSearched(false);
+            } else {
+              setSearched(true);
+              setSearchedOrders(
+                apiOrders.filter((element) => {
+                  const nameOfOrderUser = `${element.userInfo!.firstName} ${
+                    element.userInfo!.lastName
+                  }`.toLowerCase();
+                  return nameOfOrderUser.includes(value.toLowerCase());
+                }),
+              );
+            }
+          }}
```

Comment:
Can we pull this function out into `utils.js` or lift it to the container

Reply:
It's up in the container. If it is ever required elsewhere, it will be moved to `utils.js`.